js/compiler/compiler.go

```diff
@@ -37,7 +37,39 @@ import (
 
 var (
 	DefaultOpts = map[string]interface{}{
-		"presets":       []string{"latest"},
+		// "presets": []string{"latest"},
+		"plugins": []interface{}{
+			// es2015 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2015/src/index.js
+			[]interface{}{"transform-es2015-template-literals", map[string]interface{}{"loose": false, "spec": false}},
+			"transform-es2015-literals",
+			"transform-es2015-function-name",
+			[]interface{}{"transform-es2015-arrow-functions", map[string]interface{}{"spec": false}},
+			"transform-es2015-block-scoped-functions",
+			[]interface{}{"transform-es2015-classes", map[string]interface{}{"loose": false}},
+			"transform-es2015-object-super",
+			"transform-es2015-shorthand-properties",
+			"transform-es2015-duplicate-keys",
+			[]interface{}{"transform-es2015-computed-properties", map[string]interface{}{"loose": false}},
+			// "transform-es2015-for-of", // in goja
+			// "transform-es2015-sticky-regex", // in goja
+			// "transform-es2015-unicode-regex", // in goja
+			"check-es2015-constants",
+			[]interface{}{"transform-es2015-spread", map[string]interface{}{"loose": false}},
+			"transform-es2015-parameters",
+			[]interface{}{"transform-es2015-destructuring", map[string]interface{}{"loose": false}},
+			"transform-es2015-block-scoping", // let/const which particularly slow on big inputs
+			// "transform-es2015-typeof-symbol", // in goja
+			// all the other module plugins are just dropped
+			[]interface{}{"transform-es2015-modules-commonjs", map[string]interface{}{"loose": false}},
+			// "transform-regenerator", // Doesn't really work unless regeneratorRuntime is also added
+
+			// es2016 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2016/src/index.js
+			"transform-exponentiation-operator",
+
+			// es2017 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2017/src/index.js
+			// "syntax-trailing-function-commas", // in goja
+			// "transform-async-to-generator", // Doesn't really work unless regeneratorRuntime is also added
```

Comment:
Should we consider including this along with [transform-runtime](https://babeljs.io/docs/en/babel-plugin-transform-runtime) then? It would add a lot of value to support generators and this poor-man's `async`, unless they would add a lot to the bundle and explode the memory requirements.

Reply:
I think we discussed this internally and we came to the conclusion that it is better to break all the async/await stuff that is halfway working and letting people do it on their own through the [template-es6](https://github.com/k6io/template-es6) or similar.

I removed the generator tests specifically because we were failing practically all of them because it was needing `regeneratorRuntime`. And I am heavily against adding `regeneratorRuntime` and reenabling plugins, possibly bringing back the polyfill for Promise. It will likely not work great, again all of those tools are not written for k6 but for nodejs/browsers. Goja does implement Ecmascript faithfully, but there are still possibilities that those tools require something special in one of the cases to work. 

Testing this will likely show whether I am correct or not, but IMO "not async" async is not worth the time investment and explanation on why it is not working as someone would expect. And then the possibility of breaking changes once we actually start having real async.